### PR TITLE
Library update

### PR DESCRIPTION
**Pull request by tigerw**
* Updated luaexpat and expat
* Updated zlib
* Updated JsonCPP

### CODE REVIEW

**Comment by bearbin** [comment]
Why did the AppVeyor build fail?

And any other reason not to merge?

File: .gitmodules [hunk 1]
```diff
@@ -22,9 +22,16 @@
...
 	url = https://github.com/mc-server/SQLiteCpp.git
+	ignore = dirty
 [submodule "lib/libevent"]
 	path = lib/libevent
 	url = https://github.com/mc-server/libevent.git
+	ignore = dirty
+[submodule "lib/jsoncpp"]
+	path = lib/jsoncpp
+	url = https://github.com/open-source-parsers/jsoncpp.git
+	ignore = dirty
```
Comment (SphinxC0re) [inline .gitmodules]: I'm missing a newline here

**Comment by tigerw** [comment]
Question Mark™

**Comment by madmaxoft** [comment]
You couldn't make it build.

**Comment by tigerw** [comment]
No time, that is.

**Pull request closed without merging**